subsys/net/lib/lwm2m/lwm2m_engine.h

```diff
@@ -82,6 +82,13 @@ int  lwm2m_get_or_create_engine_obj(struct lwm2m_message *msg,
 				    struct lwm2m_engine_obj_inst **obj_inst,
 				    uint8_t *created);
 
+int lwm2m_engine_validate_write_access(struct lwm2m_message *msg,
+				       struct lwm2m_engine_obj_inst *obj_inst,
+				       struct lwm2m_engine_obj_field **obj_field);
```

Comment:
I wonder if we should start adding Doxygen blocks into these internal headers as well. There are quite a lot of undocumented functions. It would be easier to the developer, if there is some docs

Reply:
OK I can add.